src/index.ts

```diff
@@ -2,9 +2,7 @@ import Passage from './classes/Passage';
 export * from './types/App';
 export * from './types/AuthStrategy';
 export * from './types/JWKS';
-export * from './types/MagicLink';
 export * from './types/PassageConfig';
-export * from './types/User';
 
```

Comment:
I think we're going to need to re-export a number of types from the generated code in this file. For any user-facing function in the Passage and User class we need to make sure that any type definition used by an input or as a return value is exported up through this index.tx file. 

This is important for end users using TypeScript so they can properly build and pass around type-safe objects.

Reply:
For instance `async createMagicLink(magicLinkReq: CreateMagicLinkRequest): Promise<MagicLink>` is a function in Passage.ts so the `CreateMagicLinkRequest` and `MagicLink` types need to be exported through this index.ts from the generated code

Reply:
This actually also applies recursively - again referring to the `CreateMagicLinkRequest` interface as an example. This interface has a property `type?: CreateMagicLinkRequestTypeEnum;` so this enum definition `CreateMagicLinkRequestTypeEnum` will also need to be exported for the end user to reference in the Typescript code.

Reply:
The easiest solution might be to do something like:
`export * from './generated/models/CreateMagicLinkRequest.ts'` 
or
maybe even `export * from './generated/models'` which would definitely catch all the type defs but might be too much noise to export for the end user?

Reply:
Yes, great point. `export * from './generated/models` should be fine because the number of models will decrease once we use our actual spec.